enhancements/etcd/cluster-etcd-operator.md

```diff
@@ -490,7 +490,23 @@ items:
 
 ### Upgrade / Downgrade Strategy
 
-TODO
+#### Upgrade from 4.3 to 4.4
+
+quorum-guard: currently quorum-guard is a deployment in the `machine-config-operations` namespace the bash logic used
+to validate quorum expects certs to exist in /etc/kubernetes/static-pod-resources/etcd-member/. The target controller
+in CEO now installs these certs by revision on disk in a path such as /etc/kubernetes/static-pod-resources/etcd-pod-N/.
+
+quorum-guard was added to MCO's CVO manifests because CEO did not yet exist. Because we gracefully handle the certs in
+secrets in the `openshift-etcd` namespace it makes sense to move quorum-guard with the addition of the operator.
+
+1.) add quorum-guard deployment to CEO manifests and deploy into `openshift-etcd` namespace mounting the proper secrets.
+2.) remove quorum-guard from CVO controlled manifests in MCO repo.
```

Comment:
this is your cheap intermediate state.  I expect to see it properly managed before 4.5 ships.